### DISCUSSION TITLE
Email address associated with my previous deleted account.

### DISCUSSION THREAD

**Discussion by MuhammadZaidKhan** [community/community#51039] [GitHub Education]
Dear GitHub Support Team,

I am writing to you regarding an issue I'm facing with my GitHub account and the student pack benefits associated with it.

Previously, I had a GitHub account with the email address, which was associated with my university email. I was able to take advantage of the GitHub Student Developer Pack benefits using this account.

However, I have since deleted that account, and now I am unable to use my university email address to obtain the student benefits with my new account. When I try to apply for the student pack with my new account, I get an error message saying that my university email address is already associated with another account.

As my previous account has been deleted, I am no longer using it and I am now seeking to receive the student benefits with my new account.

I would be grateful if one could assist me in dissociating my university email address from my previous deleted account, so that I can use it with my new account to access the student benefits.

**Comment by radar-blip** [comment]
For account related issues, you could try this: [Support](https://support.github.com/contact/account)

**Comment by samus-aran** [comment]
Hi @MuhammadZaidKhan 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

We've passed this onto the internal team and when we have any updates we'll update this thread.

**Reply by MuhammadZaidKhan** [reply]
Hey any updates regarding my issue?
